scripts/setup.sh

```diff
@@ -5,6 +5,6 @@ test_mnemonic="wage thunder live sense resemble foil apple course spin horse gla
 
 persistenceCore init test --chain-id test
 echo $test_mnemonic | persistenceCore keys add test --recover --keyring-backend test
-persistenceCore add-genesis-account test 100000000000000stake --keyring-backend test
-persistenceCore gentx test 10000000stake --chain-id test --keyring-backend test
+persistenceCore add-genesis-account test 100000000000000uxprt --keyring-backend test
```

Comment:
It can have both, especially when `stake` used  to delegate.

Reply:
So should I add both here? `100000000000000uxprt` and `100000000000000stake` ?